odl/solvers/util/partial.py

```diff
@@ -108,15 +130,30 @@ class PrintIterationPartial(Partial):
 
     """Print the interation count."""
 
-    def __init__(self):
+    def __init__(self, text=None):
+        self.text = text if text is not None else 'iter'
         self.iter = 0
 
-    def send(self, _):
+    def __call__(self, _):
         """Print the current iteration."""
-        print("iter = {}".format(self.iter))
+        print("{} = {}".format(self.text, self.iter))
         self.iter += 1
 
 
+class PrintTimePartial(Partial):
```

Comment:
Went for `PrintTimingPartial` which impies that it does a timing, thought `PrintTimeDiffPartial` was somewhat long?